_metadata.yml

```
@@ -0,0 +1,14 @@
+gh:
+  org: "rostools"
+  repo: "github-intro"
+
+titles:
+  full: "An introductory workshop on using GitHub"
```

```suggestion
  full: "An introductory workshop on using GitHub for managing and working with files"
```
Maybe? So it's a bit clear what the focus of this workshop is.